### PR TITLE
Longer chat history, makes chat box transparent if chat is closed

### PR DESCRIPTION
**Pull request by Alexdoru**

### CODE REVIEW

File: src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java [hunk 1]
```diff
@@ -467,6 +470,28 @@ public LoadingConfig(File file) {
...
+                        Category.TWEAKS.toString(),
+                        "longerChat",
+                        true,
+                        "Makes the chat history longer instead of 100 lines")
+                .getBoolean();
+        chatLength = Math.max(
+                100,
+                Math.min(
+                        32767,
+                        config.get(
```
Comment (Glease) [inline src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java:483]: pretty sure forge config offers built in min max support.

File: src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java [hunk 2]
```diff
@@ -29,8 +29,13 @@ public enum Mixins {
             Side.CLIENT,
             () -> Common.config.fixPotionEffectRender,
             TargetedMod.VANILLA),
-    FIX_IMMOBILE_FIREBALLS(
-            "minecraft.MixinEntityFireball", () -> Common.config.fixImmobileFireballs, TargetedMod.VANILLA),
+    LONGER_CHAT(
```
Comment (Glease) [inline src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java:32]: Why is this removed?

**Review by Glease — CHANGES_REQUESTED** [review]

File: src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java (hunk 2 above)
Comment (Alexdoru) [reply src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java:32]: must be an error when I merge master into my chat_enhancements branch which was pretty old

File: src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java (hunk 1 above)
Comment (Alexdoru) [reply src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java:483]: I think that the mix max thing from forge is just for the in game gui thing and you can still bypass that limit by writting manually anything you want in the config file

Comment (Alexdoru) [reply src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java:483]: yeah I just tested to set this 
image

and then write 5 in the config and it acted as 5

**Review by Glease — APPROVED** [review]
